PWGJE/TableProducer/jetfinder.cxx

```diff
@@ -66,13 +66,22 @@ struct JetFinderTask {
   OutputObj<TH2F> hJetPhi{"h_jet_phi"};
   OutputObj<TH2F> hJetEta{"h_jet_eta"};
   OutputObj<TH2F> hJetN{"h_jet_n"};
+  OutputObj<TH2F> hJetRho{"h_jet_rho"};
 
   Configurable<float> vertexZCut{"vertexZCut", 10.0f, "Accepted z-vertex range"};
   Configurable<float> trackPtCut{"trackPtCut", 0.1, "minimum constituent pT"};
   Configurable<float> trackEtaCut{"trackEtaCut", 0.9, "constituent eta cut"};
-  Configurable<bool> DoRhoAreaSub{"DoRhoAreaSub", false, "do rho area subtraction"};
+
+  Configurable<bool> DoRhoAreaSub{"DoRhoAreaSub", false, "do rho area subtraction (fastjet)"};
```

Comment:
we can only select one of these at a time right? I guess the later would override each other if multiple are selected? In that case instead of having multiple configurable you can have one configurable which takes in the enum values for the subtraction types and one can set the subtraction type through that. This would cause less errors by not allowing accidentally to have multiple on at the same time

Reply:
I didn't know we can do this. I will do it ASAP.